packages/itwinui-react/src/core/Slider/Slider.tsx

```diff
@@ -152,6 +152,26 @@ export type SliderProps = {
    * Additional props for container `<div>` that hold the slider rail, thumbs, and tracks.
    */
   railContainerProps?: React.HTMLAttributes<HTMLDivElement>;
+  /**
+   * Allows props to be passed for slider-min
+   */
+  sliderMinProps?: React.ComponentProps<'span'>;
```

Comment:
lets remove `slider` prefix from all of these to make the names consistent

Reply:
fixed